infrastructure/irsa.tf

```diff
@@ -0,0 +1,31 @@
+module "vpc_cni_irsa_role" {
```

Comment:
You don't need this for VPC CNI, you will install CNI via **cluster_addons** in eks module.